src/histolab/util.py

```diff
@@ -206,6 +204,7 @@ def threshold_to_mask(
     np.ndarray
         Boolean NumPy array representing a mask where a pixel has a value True
         if the corresponding input array pixel exceeds the threshold value.
+        if the corresponding input array pixel exceeds the threshold value.
```

Comment:
?

Reply:
there was a blank line, i don't know why github is marking this change this way